packages/pigeon/CHANGELOG.md

```diff
@@ -1,12 +1,10 @@
-## 5.0.0
+## 5.0.1
 
-* Creates new Generator classes for each language.
+* [kotlin] Fixes a bug with a missed line break between generated statements in the `fromList` function of the companion object.
 
-## 4.2.16
+## 5.0.0
 
-* [swift] Fixes warnings with `Object` parameters.
-* [dart] Fixes warnings with `Object` return values.
-* [c++] Generation of APIs that use `Object` no longer fails.
```

Comment:
You've accidentally deleted the changelog for 4.2.16 =)

Reply:
Ups, restored 😉